tools/i18n/makepot.php

```diff
@@ -68,14 +68,14 @@ class MakePOT {
 		),
 		'cp-plugin' => array(
 			'description' => 'Translation of the ClassicPress plugin {name} {version} by {author}',
-			'msgid-bugs-address' => 'https://wordpress.org/support/plugin/{slug}',
+			'msgid-bugs-address' => 'https://forums.classicpress.net/plugin/{slug}',
 			'copyright-holder' => '{author}',
 			'package-name' => '{name}',
 			'package-version' => '{version}',
 		),
 		'cp-theme' => array(
 			'description' => 'Translation of the ClassicPress theme {name} {version} by {author}',
-			'msgid-bugs-address' => 'https://wordpress.org/support/theme/{slug}',
+			'msgid-bugs-address' => 'https://forums.classicpress.net/theme/{slug}',
```

Comment:
This change is incorrect as we do not have subforums for each theme.

Reply:
Let's leave it pointing to WP for now - we're using their theme directory, makes sense to leave support there.

Reply:
The description should probably still say WordPress too.